### ISSUE TITLE
Houdini Solaris Support

### ISSUE THREAD

**Issue by parsaiej** [Autodesk/Aurora#34]
Hi, thanks for this project.

I am wondering if there are plans to make it easier to produce a USD render delegate that is compatible with a Houdini Solaris viewport.

In order to do this, I am pretty sure you just need to link against the USD libs that ships with a given Houdini version.

**Comment by pixnblox** [comment]
The project needs to be built with a full build of OpenUSD, including the binaries and header files. Also, OpenUSD is changing rapidly, so there are likely changes (hopefully small) that need to be made to the render delegate to compile.

Note that for Vulkan Ray Tracing, we currently need a specific Autodesk branch of OpenUSD that has ray tracing support in OpenUSD's Hgi interface. The DirectX Raytracing implementation will work with a "vanilla" OpenUSD release.

So the first step would be to determine which OpenUSD release (or commit) is being used by your version of Houdini, then try to build Aurora against a build of that OpenUSD release. I hope that makes sense, and let us know if you need more info!

**Comment by adro79** [comment]
Hi, I was wondering the same. Would be cool to be able to implement this renderer into DCCs.

As @pixnblox says, to build Aurora with Vulkan you should build a specific branch of USD first.

But if you're using Windows, you can build Aurora with DirectX. To use the Houdini USD libs you should tell cmake where to look using CmakeFiles, Dreamworks has made some to be able to build Moonray for Houdini, you can look [my guide](https://github.com/adro79/openmoonray/discussions/2) as an example.

I take the opportunity to ask, is there any plan to progress with the Aurora branch of USD? That version is quite old and would be nice to be able to use a more modern USD version.

**Comment by pixnblox** [comment]
> I take the opportunity to ask, is there any plan to progress with the Aurora branch of USD? That version is quite old and would be nice to be able to use a more modern USD version.

Yes, we do plan to update the OpenUSD version. It may be relatively easy to update to a specific version that you may need (often the case with DCCs), but we were thinking of official moving to 24.08, which will be the next major release.

**Comment by pixnblox** [comment]
FYI, we have updated to OpenUSD 24.08 with our latest release.
